.github/workflows/ci-pkg-install.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         if: ${{ matrix.pkg-name == 'lightning' || matrix.pkg-name == 'notset' }}
         run: python -m lightning --version
       - name: Run CLI (direct bash)
-        if: ${{ matrix.pkg-name == 'lightning' || matrix.pkg-name == 'notset' || matrix.pkg-name == 'app' }}
+        if: ${{ matrix.pkg-name == 'lightning' || matrix.pkg-name == 'notset' }}
         run: lightning --version
```

Comment:
Does this change mean that lightning_app doesn't include the CLI? Wouldn't we wanto them to go together? Similarly to how PL includes fabric

Reply:
example of going together similar as Pl + Fabric is good one, let me double check it